README.md

```diff
@@ -1,100 +1,137 @@
 
+   ```sh
+   cd ostis-web-platform/scripts/
+   ./prepare.sh
+   ```
 
-## Natively
-Note: Currently only Ubuntu is supported by this installation method. If you're going to use it, all build-time dependencies will be downloaded to your computer, and it might take a while to compile the `sc-machine`. Use it only if you know what you're doing!
-```sh
-cd ostis-web-platform/scripts/
-./prepare.sh
-```
+## Usage
 
-# Usage
-## Docker
-```sh
-# This command will build the Knowledge Base.
-# You need to launch it only before the first launch (or if you've made updates to KB sources)
-docker compose run machine build
-# This will start platform services and run web interface at localhost:8000
-docker compose up
-```
++ Docker Compose
 
-## Native installation
-After run the following script for sc-server to interact with knowledge processing machine:
-```sh
-cd ostis-web-platform/scripts/
-./run_sc_server.sh
-```
+   ```sh
+   # build the Knowledge Base.
+   # Required before the first startup (or if you've made updates to KB sources)
+   docker compose run machine build
+   # start platform services and run web interface at localhost:8000
+   docker compose up
+   ```
 
-To see and interact you with system run the following script for interface:
-```sh
-cd ostis-web-platform/scripts/
-./run_scweb.sh
-```
++ Native installation
+
+   ```sh
+   # Build knowledge base
+   cd ostis-web-platform/scripts/
+   ./build_kb.sh
+   # Launch knowledge processing machine
+   cd ostis-web-platform/scripts/
+   ./run_sc_server.sh
+   # *in another terminal*
+   # Launch semantic web interface at localhost:8000
+   cd ostis-web-platform/scripts/
+   ./run_scweb.sh
+   ```
+
+## Documentation
+
+We document all information about project development and its components' implementation in sources of its knowledge base,
+to provide opportunity to use it in information processing and knowledge generation.
+
+You can access the current version of the documentation in [docs/main.pdf](docs/main.pdf) file of this project.
 
-# Documentation on SC-code representation
-We represent all information about project development and its components' implementation in sources of its knowledge 
-base to provide opportunity to use it in information processing and knowledge creating. Documentation is written with 
-the help of LaTex tools in SCn-code representation. To see more information about [the OSTIS Technology](https://github.com/ostis-ai/ostis-standard) 
-web-oriented platform implementation:
+Documentation is written with
+the help of LaTeX tools in SCn-code representation. To build documentation manually, you'll need a LaTeX distribution installed on your computer. Alternatively we're providing a Docker image to build the documentation in case you can't / don't want to install LaTeX on your PC.
```

Comment:
```suggestion
the help of LaTeX tools in SCn-code representation. To build documentation manually, you'll need a LaTeX distribution installed on your computer. Alternatively, we provide a Docker image to build the documentation in case you can't / don't want to install LaTeX on your PC.
```